cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -185,31 +175,10 @@ data LiveViewState blk a = LiveViewState
     , lvsMetricsThread        :: !LiveViewThread
     , lvsNodeThread           :: !LiveViewThread
 
-    , lvsNodeKernel           :: !(SMaybe (LVNodeKernel blk))
-    , lvsPeersLastTime        :: !Word64
     , lvsPeers                :: [LVPeer blk]
     , lvsColorTheme           :: !ColorTheme
     } deriving (Generic, NFData, NoUnexpectedThunks)
```

Comment:
How large is this record (like C's `sizeof`, not including the things it points to)? Each time you modify it, you make a copy of the *entire record*. Depending on how often it is updated, this might be putting extra pressure on the GC.

You could group related values into sub records (e.g, if all the disk usage fields are read together, group them in a single record), then the number of bytes to copy shrinks. For example, grouping 10 `Word64` in one record will shrink the size of `LiveViewState` by 9 * 8 bytes.

Reply:
> You could group related values into sub records..

Agree. It will be done in another PR.